src/modules/rating/controllers/rating.controller.ts

```diff
@@ -0,0 +1,39 @@
+import { Body, Controller, Logger, Param, Post } from '@nestjs/common'
+import { RatingRepository } from '../database/rating.repository'
+import {
+	CreateRatingRequestDTO,
+	CreateRatingResponseDTO,
+} from './dtos/create-rating.dtos'
+import { ApiResponse, ApiTags } from '@nestjs/swagger'
+import { Rating } from '../database/rating.model'
+import { ObjectIdParam } from '@modules/admin/product/controllers/dtos/common.dto'
+
+@Controller('v1/products')
+@ApiTags('Rating')
+export class RatingController {
+	private readonly logger: Logger = new Logger(RatingController.name)
+	constructor(private readonly ratingRepo: RatingRepository) {}
+	@Post('/:id/ratings')
```

Comment:
Please follow the RESTful API convention as following:

- create rating: POST /api/v1/ratings

- get list ratings: GET /api/v1/ratings

- update a rating: PUT /api/v1/ratings/:id

- delete a rating: DELETE /api/v1/ratings/:id